### DISCUSSION TITLE
How to setup an IRC server to control the game console remotely?

### DISCUSSION THREAD

**Discussion by joaoglibras** [piqueserver/piqueserver#740] [Q&A]
I don't even know if the IRC have this purpose. But, as I've said, I would like to use the IRC to control my server. If there is any better solution to do the same thing, I accept it.

Here is what I've already did:
-Port forwarding is alright, the IRC server is connected to quakenet and working fine.
-When I type "say something" in the piqueserver console, the message shows in the IRC chat.

My problem is: how do I say something as my server through the IRC chat? Without using the piqueserver prompt to do do it.

Any help will be appreciated!

**Comment by NotAFile** [comment]
I don't remember the details, but there is a command prefix and you are require to be OP in the channel for things to work

**Comment by joaoglibras** [comment]
I know the prefixes, it's "." and "!". But that's what I'm struggling to get right: how do I enter as an OP in the channel? I can't enter in the IRC as OP.

**Reply by NotAFile** [reply]
You'll have to claim the channel with the IRC network, how to do that depends on the network
